tools/revng-lift/BinaryFile.cpp

```diff
@@ -1002,7 +1011,13 @@ void BinaryFile::parseELF(object::ObjectFile *TheBinary,
                                    - u64(ProgramHeader.p_offset));
         ProgramHeaders.Address = PhdrAddress;
       }
+
+      if (pagebuster) {
+        g_epoch_counter++;
+      }
```

Comment:
Maybe we should outline the whole body of this `case` into a function and call it differently depending on whether we're in `case PT_PAGEBUSTER` or `case PT_LOAD`.